A2rchi/interfaces/chat_app/app.py

```diff
@@ -258,21 +392,24 @@ def like(self):
             data = request.json
 
             # Extract the HTML content and any other data you need
-            chat_content = data.get('content')
-            discussion_id = data.get('discussion_id')
             message_id = data.get('message_id')
 
             feedback = {
-                "chat_content" :  chat_content,
-                "message_id"   :  message_id,
-                "feedback"     :  "like",
+                "message_id"   : message_id,
+                "feedback"     : "like",
+                "feedback_ts"  : datetime.now(),
```

Comment:
Why use datetime instead of time.time()? Just want to make sure we stay consistent across the repo with how we talk about timestamps.